src/ansys/fluent/core/services/reduction.py

```diff
@@ -300,7 +300,7 @@ def centroid(self, locations, ctxt=None) -> Any:
         request = ReductionProtoModule.CentroidRequest()
         request.locations.extend(self._get_location_string(locations, ctxt))
         response = self.service.centroid(request)
-        return response.value
+        return Vector(response.value)
```

Comment:
Question: has returning a numpy array or a simple list been considered? Why return a custom class `Vector` instead of one of the standard types?

Reply:
Good point @raph-luc. Will surely look into it.

Reply:
I think returning a tuple of reals is better, which will be consistent with settings API also.

Reply:
Done, Thank you.